woocommerce/woocommerce-gutenberg-products-block
# Convert `Handpicked Products` block to TS

Converts the `Handpicked Products` block.js file to TS.

### Testing

#### Automated Tests

### Manual Testing

1. Create a page and add a `Handpicked products` block.
2. Save the page and check it renders correctly the default config in the frontend.
3. Edit the page again and make some changes to the block (hiding some content, changing columns number, ordering, etc.).
4. Save the page and check it renders properly with the new config.
